src/main/java/functions/TaskList.java

```diff
@@ -0,0 +1,93 @@
+package functions;
+
+import datatypes.Deadline;
+import datatypes.Event;
+import datatypes.Task;
+import datatypes.Todo;
+
```

Comment:
I think this blank line can be removed to make the code more elegant

Reply:
Resolved. Thank you!